content/blog/getting-started-with-spark-on-mapr-sandbox.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Spark on MapR Sandbox
+date: 2021-10-14T06:01:00.000Z
+author: MapR Tutorials
+authorimage: /img/Avatar1.svg
+---
+
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+At MapR, we distribute and support Apache Spark as part of the MapR Converged Data Platform. This tutorial will help you get started with running Spark applications on the MapR Sandbox.
+
+## Prerequisites
+
+**HARDWARE REQUIREMENTS**
+
+* 8GB RAM, multi-core CPU
+* 20GB minimum HDD space
+* Internet access
+
+**SOFTWARE REQUIREMENTS**
+
+* A hypervisor. This example uses VMware Fusion 6.0.2 on OSX; however, other VMware products could be used instead. Additionally, [VirtualBox](https://www.virtualbox.org/) can be used
+* A virtual machine image for the MapR Sandbox. Spark comes pre-loaded with version 5.0 and later of the MapR Sandbox.
+
+**Starting up and Logging into the Sandbox**\
+Install and fire up the Sandbox using the instructions here: <https://docs.datafabric.hpe.com/62/MapRContainerDevelopers/RunMapRContainerDevelopers.html>. Once you are able to login to the web interface for the Sandbox, you are ready to start setting up Spark.
+
+**Logging in to the Command Line**
+
+* Before you get started, you'll want to have the IP address handy for your Sandbox VM. See the screenshot below for an example of where to find that.  
+
+  ![](/img/tutorial_spark1.png)
+* Next, use an SSH client such as Putty (Windows) or Terminal (Mac) to login. See below for an example:\
+  use userid: user01 and password: mapr.  
+
+  ![](/img/tutorial_spark2.png)
+
+  For VMWare use: `$ ssh user01@ipaddress`
+
+  For Virtualbox use: `$ ssh user01@127.0.0.1 -p 2222`
+
+## “How to” for a Spark Application
+
+Next, we will look at how to write, compile, and run a Spark word count application on the MapR Sandbox. First we will walk step by step through the following word count application in Java.
+
+**Example Word Count App in Java**
+
+![](/img/tutorial_spark3.png)
```

Comment:
See if you can make the image smaller so it's not quite so fuzzy
![image](https://user-images.githubusercontent.com/77496261/137520223-d8b3c36e-75e1-463b-b711-56baab35ec51.png)